src/toil/fileStore.py

```diff
@@ -851,7 +851,8 @@ def cleanCache(self, newJobReqs):
                 #                  (self.decodedFileID(cachedFile), cachedFileSize))
                 logger.debug('CACHE: Evicted  file with ID \'%s\' (%s bytes)' %
                              (self.decodedFileID(cachedFile), cachedFileSize))
-            assert cacheInfo.isBalanced(), 'Unable to free up enough space for caching.'
```

Comment:
Hannes and Arjun -- thanks for the additional information on this and the pointer to the latest update. I pulled in #1052 and it does fix the problem running the tests. Brilliant. I reverted the change and have a local version passing all of the CWL v1.0 tests. As soon as common-workflow-language/common-workflow-language#268 gets merged I can update to that and will push the final CWL fixes needed. Thanks again for the help.